packages/amplify-category-api/src/index.ts

```diff
@@ -308,12 +308,6 @@ export async function transformCategoryStack(context: $TSContext, resource: $TSO
     }
   } else if (resource.service === AmplifySupportedService.APIGW) {
     if (canResourceBeTransformed(resource.resourceName)) {
-      const backendDir = pathManager.getBackendDirPath();
-      const overrideDir = pathManager.getResourceDirectoryPath(undefined, AmplifyCategories.API, resource.resourceName);
-      await buildOverrideDir(backendDir, overrideDir).catch(error => {
```

Comment:
Why remove this?

Reply:
we are doing buildOverride inside transform , same as other categories except for graphql